udemy_exercises/bank/Bank.java

```diff
@@ -1,9 +1,188 @@
 package udemy_exercises.bank;
 
+import java.util.*;
+
 public class Bank {
-    
-}
 
+  ArrayList<Branch> branchList = new ArrayList<Branch>();
+  Scanner s, sc;
+
+  public void menu() {
+
+    s = new Scanner(System.in);
+    sc = new Scanner(System.in);
+    int option = 0;
+
+    while (option != 99) {
+
+      System.out.println("\nPlease enter in an option from the menu:");
+      System.out.println(
+          "\t1. Add Branch\n\t2. Add Customer to Branch\n\t3. Add Transaction for Customer in a Branch\n\t4. Show Branches\n\t5. Show Branch Customers\n\t99. Quit");
+
```

Comment:
I like that you went and added a console menu, but you should validate your input so your program doesn't crash if your user gives you an invalid type